### DISCUSSION TITLE
Is is possible to implement a custom functionality?

### DISCUSSION THREAD

**Discussion by atrolex** [atrocore/atrocore#259] [Q&A] [answered]
I want to have some custom functionality which is currently not available.

**Comment by atrolex** [comment, marked as answer]
You can create your own modules to implement a custom functionality. Here is the Guide [How to Create Own Module](https://help.atrocore.com/#/atrocore/developer-guide/creating-own-module).

**Alternatively** you can order the custom development from our team, please contact our support for that. In case the functionality you need will be included in the core or some available module, we will give your 50% on the development cost.

**Reply by ahmed26399** [reply]
Hey,
I want to know if i order the paid custom development from the team, the custom functionality  can be reversed in Open source ?
thanks in advance

**Reply by atrolex** [reply]
If you want something useful for the others, we give you 50% discount on the development cost and the feature will be implemented as a standard software. You can email us your request to team@.
